detection-rules/attachment_office_remote_doc_template.yml

```
@@ -0,0 +1,24 @@
+name: "Attachment: Office document loads remote document template"
+description: |
+  Recursively scans archives and Office documents to detect remote document template injection.
+references:
+  - "https://delivr.to/payloads?id=c7a7195e-0de3-428d-a32c-5fd59a3012da"
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and any(attachments,
+      (
+          (
+          .file_extension in~ $file_extensions_macros 
+          or .file_extension in~ $file_extensions_common_archives
```

```suggestion
              // office files
              .file_extension in~ ("doc", "docm", "docx", "dot", "dotm", "pptm", "ppsm", "xlm", "xls", "xlsb", "xlsm", "xlt", "xltm")
              or .file_extension in~ $file_extensions_common_archives
```

templates can be loaded from non-macro enabled office files, so this might be more desirable. we have an example of this usage in this rule: https://github.com/sublime-security/sublime-rules/blob/1a5c6923ba6f61c7b8b1e6fae22bb669af192502/detection-rules/attachment_office_remote_html_template.yml#L31